tests/old_tests/issue5/relimport.py

```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8; -*-
 # See issue #5
 
-from .mod2 import foo
+from . import mod1, mod1 as moo
```

Comment:
This version gives just one import statement to analyze, rather than the two in the original. Note that they trigger different code paths. Maybe we need to:

```python
from . import mod1
from . import mod4 as moo
```

or something similar.

Reply:
fixed with #noqa statement

Reply:
That's even better. Thanks.